qiskit_experiments/characterization/qubit_spectroscopy.py

```diff
@@ -343,6 +343,7 @@ def __init__(
         frequencies: Union[List[float], np.array],
         unit: Optional[str] = "Hz",
```

Comment:
Not part of this PR, but I notice some hand-wired unit conversion code in the spectroscopy classes.  If we continue to allow non-SI units this seems likely to be part of almost every experiment class -- should we pick a unit conversion library?  (Otherwise it seems likely we'll end up with each class accepting slightly different sets)

Reply:
How bout like so de9daf0 ?